src/main/java/org/opensearch/knn/training/TrainingJobClusterStateListener.java

```diff
@@ -112,7 +112,7 @@ protected void updateModelsNewCluster() throws IOException, InterruptedException
                 Model model = modelDao.get(modelId);
```

Comment:
We can remove this line.

Reply:
Per our discussion offline, I changed the line and refactored the code to ensure backwards compatibility.